k8s/base.py

```diff
@@ -207,6 +186,37 @@ def save(self):
             resp = self._client.put(url, self.as_dict())
         self.update_from_dict(resp.json())
 
+    def _label_selector(labels):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [first argument of a method should be named 'self' (N805)](https://app.codacy.com/manual/fiaas/k8s/pullRequest?prid=5606786)